doc/cookbook/source/examples/clustering/gmm.rst

```diff
@@ -0,0 +1,53 @@
+=======================
+Gaussian Mixture Models
+=======================
+
+A Gaussian Mixture Model is a probabilistic model that assumes that data are generated from a finite mixture of Gaussians with unknown parameters. The model likelihood can be written as:
+
+.. math::
+
+    p(x|\theta) = \sum_{i=1}^{K}{\pi_i N(x|\mu_i, \Sigma_i)}
+
+where :math:`p(x|\theta)` is probability distribution given :math:`\theta:=\{\mu_i, \Sigma_i\}_{i=1}^K`, :math:`K` denotes number of mixture components, :math:`\pi_i` denotes weight for :math:`i`-th component, :math:`N` denotes a multivariate normal distribution with mean vector :math:`\mu_i` and covariance matrix :math:`\Sigma_i`.
```

Comment:
Can you use `\mathcal{N}` for the Gaussian (maybe misunderstood me earlier, I just wanted the x in there)